.github/workflows/update_version.yml

```diff
@@ -8,9 +8,6 @@ on:
     inputs:
       project_version:
         description: "Version number to be released e.g 10.0.3"
-      postgres_version:
```

Comment:
This one is additional. I had removed postgres version update long time ago. I removed here in the pipeline as well to be able to execute update_docker from Actions pipeline screen